dumbpipe-web/Cargo.lock

```diff
@@ -789,7 +789,7 @@ dependencies = [
  "anyhow",
  "clap",
  "hex",
- "iroh-net",
+ "iroh-net 0.14.0",
```

Comment:
this still has an `0.14` dependency?

Reply:
???? sigh, i probably accidentally did a key stroke or something

Reply:
note: this is a lock file, this just means sth in the dep tree still depends on 0.14.0

Reply:
yea, we need to wait until the latest dumbpipe is released on crates.io , created an issue #51